### ISSUE TITLE
Base64 Encoded Images Not Showing in Email Clients

### ISSUE THREAD

**Issue by ameerasghar** [mautic/mautic#12360]
### Mautic Version

4.4.x series

### PHP version

7.4

### What browsers are you seeing the problem on?

Not relevant

### What happened?

Base64 encoded images don't show in email clients.

The images are showing fine in Mautic preview. But when the image is sent to an email address, and viewed in an email client, we can only see alt text for the image.

Ex., of images in the Mautic Preview screen:
![Screenshot 2023-05-17 at 3 27 46 PM](https://github.com/mautic/mautic/assets/46496592/97e0044f-8a24-4c63-8fac-da11365be86c)

How images show in the email client:
![Screenshot 2023-05-17 at 3 28 23 PM](https://github.com/mautic/mautic/assets/46496592/3e496911-b77c-4ea8-984f-3933229fa08b)

How the source code looks for the email (download source in the email client). There is no source of the images:
![image](https://github.com/mautic/mautic/assets/46496592/fc3e0ab1-9741-4a33-b6d0-4d08b11b17ea)

We have temporarily circumvented this issue by turning off Base64 encoding. But we would prefer to have Base64 encoding for better email performance

### How can we reproduce this issue?

Step 1: Turn on Base64 encoding for Mautic
Step 2: Create an email with at least 1 image
Step 3: Preview the email to ensure the image is showing
Step 4: Send the email to yourself as a test, and the images will not show

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by ameerasghar** [comment]
@gabepri

**Comment by ameerasghar** [comment]
@incentfit

**Comment by dsp76** [comment]
Not a good idea to use base64 encoded images in mails. At least Gmail and Windows based Outlook don't support them at all:
https://www.caniemail.com/features/image-base64/

Maybe the once that seem to be "broken" don`t support them at all?

**Comment by ameerasghar** [comment]
Base 64 images were working for us up until May 26, 2022 (see below)
![Screenshot 2023-06-14 11 52 23 AM](https://github.com/mautic/mautic/assets/46496592/d1317bf1-7db2-4c72-b87b-4b1034fa12e6)

As of 2023, it seems Base 64 images from Mautic are broken (same email header, images are broken now):
![Screenshot 2023-06-14 11 52 52 AM](https://github.com/mautic/mautic/assets/46496592/82df0ca3-c7cf-4c70-a040-fdfc1a2e2b30)

**Comment by dsp76** [comment]
Which client are you testing with? Do you use the legacy editor or grape js?

**Comment by ameerasghar** [comment]
@dsp76 sorry for the late response.

We tested with Google Mail. You brought up a good point about type of editor.

We're still using the legacy editor, and are not using grape js.

The only difference is that we switched the Mautic Theme from the Vibrant theme (see first image below) to Blank theme (see second image below)

<img width="269" alt="Screenshot 2023-06-28 at 12 22 20 PM" src="https://github.com/mautic/mautic/assets/46496592/69492fbc-9ac7-4061-a60d-6ea74c19a9f2">

<img width="279" alt="Screenshot 2023-06-28 at 12 22 57 PM" src="https://github.com/mautic/mautic/assets/46496592/b83729ff-0749-4931-9143-5a84968eed72">

But we also turned on Base 64 encryption last year.
